chirp/drivers/icf.py

```diff
@@ -505,6 +505,9 @@ def _clone_to_radio(radio):
             LOG.debug("Waiting for clone result...")
             time.sleep(0.5)
 
+    if radio.ignore_clone_result() is True:
```

Comment:
No need for the `is True`.. Some linters will complain if you do this, and some only if you compare with `==` but either way, please just test the truthiness, don't require it to be exactly `True`.